integration_tests/backends/itt/src/itt_example_c/itt_example.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <ittnotify.h>
+
+int main(void) {
+    __itt_domain* domain = __itt_domain_create("Example.Domain");
+    __itt_string_handle* task1 = __itt_string_handle_create("Task 1");
+    __itt_string_handle* task2 = __itt_string_handle_create("Task 2");
+
+    printf("Starting ITT example...\n");
+
+    __itt_task_begin(domain, __itt_null, __itt_null, task1);
+    for (volatile int i = 0; i < 100000000; ++i);
```

Comment:
same as in python, maybe no need any loop : )

Reply:
Will remove loop.